Palindrome.java

```diff
@@ -0,0 +1,132 @@
+package WEEK1;
+
+import static org.testng.Assert.assertEquals;
+
+//import java.util.Scanner;
+
+import org.junit.Test;
+
+public class Palindrome {
+	static int length = 0;
+	
+	public static boolean is_palindrome(Node first, Node second){
```

Comment:
is_palindrome and palindrome are basically the same name and it would be hard to tell which to use in what situation.  It looks like this function is checking whether the two lists are equal, so maybe something like AreEqual or something like that